src/utils/kubeclient.ts

```diff
@@ -137,6 +139,38 @@ export const getKubeAccess = async (namespace: string, currentContext: string):
   };
 };
 
+export const startKubeProxy = (listener: (...args: any[]) => void, port: number) => {
+  if (isRendererThread()) {
+    runCommandStreamInMainThread(listener, {
+      commandId: uuid(),
+      cmd: 'kubectl',
+      args: ['proxy', '--port', `${port}`],
+    });
+  } else {
+    runCommandStream(listener, {
+      commandId: uuid(),
+      cmd: 'kubectl',
+      args: ['proxy', '--port', `${port}`],
+    });
+  }
+};
+
+export const killKubeProxy = (listener: (...args: any[]) => void) => {
+  if (isRendererThread()) {
+    runCommandStreamInMainThread(listener, {
+      commandId: uuid(),
+      cmd: 'pkill',
```

Comment:
will this work on Windows as well?

Reply:
nope, I commented a comment on slack, but couldn't try it yet